src/styles/blocks/stars.css

```diff
@@ -0,0 +1,42 @@
+.stars_container {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-image: url(../../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1) {
+  background-image: url(../../images/star-active.svg);
+}
+
+.stars--2 .stars__star:nth-child(-n + 2) {
+  background-image: url(../../images/star-active.svg);
+}
+
+.stars--3 .stars__star:nth-child(-n + 3) {
+  background-image: url(../../images/star-active.svg);
+}
+
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background-image: url(../../images/star-active.svg);
+}
+
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url(../../images/star-active.svg);
+}
```

Comment:
The use of `stars--N` modifiers with pseudo-selectors to change the background image is not the best approach for dynamic state changes. Instead, consider using JavaScript to add or remove classes based on user interaction to dynamically update the star rating. This will allow for more flexibility and maintainability.